conda_forge_tick/migrators/version.py

```diff
@@ -197,21 +221,34 @@ def migrate(
         **kwargs: Any,
     ) -> "MigrationUidTypedDict":
         version = attrs["new_version"]
-
-        with open(os.path.join(recipe_dir, "meta.yaml")) as fp:
-            raw_meta_yaml = fp.read()
-
-        updated_meta_yaml, errors = update_version(
-            raw_meta_yaml,
-            version,
-            hash_type=hash_type,
-        )
+        recipe_dir = Path(recipe_dir)
+        recipe = None
+        recipe_v0 = recipe_dir / "meta.yaml"
+        recipe_v1 = recipe_dir / "recipe.yaml"
+        if recipe_v0.exists():
```

Comment:
Please use the same name as in other parts of the implementation. I think `recipe` is too ambiguous anyway, so it's preferable to match the already existing use of `recipe_path` here, rather than vice versa.
```suggestion
        recipe_path = None
        recipe_path_v0 = recipe_dir / "meta.yaml"
        recipe_path_v1 = recipe_dir / "recipe.yaml"
        if recipe_path_v0.exists():
```